ci-operator/step-registry/ovn/sdn-migration-rollback/ovn-sdn-migration-rollback-commands.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TARGET=${TARGET:-OpenShiftSDN}
+
+oc patch MachineConfigPool master --type='merge' --patch '{"spec":{"paused":true}}'
+oc patch MachineConfigPool worker --type='merge' --patch '{"spec":{"paused":true}}'
+
+# Reset the spec.migration before we can set it to other value
+oc patch Network.operator.openshift.io cluster --type='merge' --patch '{"spec":{"migration":null}}'
+# Wait until CNO update the applied-cluster cm to the latest
+sleep 10
+
+# Allow rollback
+# Change network to target network in Network.config.openshift.io the CR to trigger machine config update by MCO.
+oc patch Network.operator.openshift.io cluster --type='merge' --patch "{\"spec\":{\"migration\":{\"networkType\":\"${TARGET}\"}}}"
+oc patch Network.config.openshift.io cluster --type='merge' --patch "{\"spec\":{\"networkType\":\"${TARGET}\"}}"
+
+sleep 30
```

Comment:
Same, use `oc wait --for` for operator to stop updating

Reply:
perhaps waiting for CNO to start Progressing=True and then waiting for Progressing=False would work?

Reply:
fixed